app/services/billing-status-display.ts

```diff
@@ -20,7 +20,7 @@ export default class BillingStatusDisplayService extends Service {
     );
   }
 
-  get shouldShowSubscribeButton(): boolean {
+  get shouldShowUpgradeButton(): boolean {
     return !!this.currentUser && !this.currentUser.canAccessPaidContent && this.router.currentRouteName !== 'pay';
```

Comment:
To reflect the new logic, I'd change this to return `true` is there's no authed user,  something like: 

```
if (!this.currentUser) {
    return true;
}
```